run.sh

```diff
@@ -52,7 +52,11 @@ if [ $INITIALIZE_TOOL_DEPENDENCIES -eq 1 ]; then
     python ./scripts/manage_tool_dependencies.py -c "$GALAXY_CONFIG_FILE" init_if_needed
 fi
 
-if [ -n "$GALAXY_RUN_ALL" ]; then
+if [ -n "$GALAXY_UWSGI" ]; then
+    uwsgi_args="$(python ./scripts/get_uwsgi_args.py)"
```

Comment:
Did you review common_startup_functions.sh add this release meant to synchronize launching via uwsgi across reports, tool shed, and Galaxy? If no, can you and then comment on whether it makes sense and if we can move get_uwsgi_args.py into that template and share those functions.

I'm working on writing some tests for this PR this morning, if I can't get those done I'll merge this PR this afternoon.

Reply:
I'm taking a look at this.

Reply:
Related, I wonder if the new `galaxy.script` I added should move to `galaxy.util.properties`. I think the only reason I didn't put it in properties was because of all the 3rd party imports for `galaxy.util`?

Reply:
Maybe because of the globals in `galaxy.script` - which was a way to avoid having to modify the scripts that already use these functions the way they do. Probably could just fix that. But maybe `galaxy.util.script` makes more sense anyway.